### PR TITLE
Feature#1881 mejs4 add to playlist

### PR DESCRIPTION
**Pull request by adamjarling**
Let me know if there's any questions on anything.  Note there are no tests.  Do we do application tests for views?   Nothing changed in the controller or model.

### CODE REVIEW

**Comment by adamjarling** [comment]
I'm assuming that whoever merges can squash all the commits into one merge commit. If you'd prefer I squash these locally then push, just let me know.

File: app/assets/javascripts/media_player_wrapper/avalon_player_new.es6 [hunk 1]
```diff
@@ -226,15 +242,19 @@ class MEJSPlayer {
     let defaults = {
       alwaysShowControls: true,
       pluginPath: "/assets/mediaelement/shims/",
-      features: ['playpause', 'current', 'progress', 'duration', 'volume', 'quality', 'fullscreen'],
+      features: ['playpause', 'current', 'progress', 'duration', 'volume', 'quality', 'fullscreen', 'addToPlaylist'],
```
Comment (cjcolvar) [inline app/assets/javascripts/media_player_wrapper/avalon_player_new.es6]: Please put addToPlaylist before fullscreen.  The order of this array indicates the order the controls appear and we always want fullscreen on the far right.

Comment (adamjarling) [reply app/assets/javascripts/media_player_wrapper/avalon_player_new.es6]: Got it.  Updating now.

**Comment by adamjarling** [comment]
@cjcolvar I made your updates to this PR, squashed all my commits locally, and just pushed up the branch.  Let me know if all looks ok.

**Comment by lealeelu** [comment]
Looks good to me.

**Comment by adamjarling** [comment]
@lealeelu Yes, to load MEJS4, append `?mejs4=true` to the end of a player url.

**Pull request merged**